<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUpdateDeleteUV</class>
 <widget class="QDialog" name="DialogUpdateDeleteUV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="uvsModelCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateUV">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="updateUVverticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="code">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="codeLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="titre">
          <property name="text">
           <string>Titre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titreLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nombre de crédits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nbCreditspinBox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="saison">
          <property name="text">
           <string>Saison</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="saisonComb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="categorie">
          <property name="text">
           <string>Catégorie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="categorieComb"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="confirmUpdate">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
